pfdp/css/stats.css

```diff
@@ -0,0 +1,35 @@
+.stat-container {
+  display: flex;
+  justify-content: space-around;
+}
+
+.stat-card {
+  /* color: grey; */
+  width: 28%;
+}
+
+.stat-number {
+  font-family: "Montserrat", sans-serif !important;
+  font-size: 70px !important;
```

Comment:
`! important` not necessary here, you can delete it

Reply:
see above